static/src/stylesheets/module/identity/_wrapper.scss

```diff
@@ -24,7 +24,7 @@
 .identity-wrapper {
     margin-left: auto;
     margin-right: auto;
-    max-width: gs-span(6);
```

Comment:
IIRC this is gonna affect the consents journey :( does it still look okay at this size?

Reply:
you're right, still looks alright I think

![image](https://user-images.githubusercontent.com/8754692/40480660-13e39704-5f47-11e8-8108-51e1a612beaf.png)

Reply:
just grepped `identity-wrapper` and it turns out this is used extensively. Is having a `identity-wrapper--wide` class only for mma an option? I'd prefer that to suddenly widening up most identity pages